keyboards/keebio/iris/readme.md

```diff
@@ -3,18 +3,36 @@ Iris
 
 A split ergo 4x6 keyboard with 3 or 4 thumb keys made and sold by Keebio. [More info at Keebio](https://keeb.io).
 
-Keyboard Maintainer: [Bakingpy/nooges](https://github.com/nooges)  
-Hardware Supported: Pro Micro  
-Hardware Availability: [Keebio](https://keeb.io)  
+Keyboard Maintainer: [Bakingpy/nooges](https://github.com/nooges)
```

Comment:
The "Keyboard Maintainer" and "Hardware Supported" lines need to keep the trailing double spaces, in order to render properly on GitHub.

```suggestion
Keyboard Maintainer: [Bakingpy/nooges](https://github.com/nooges)  
```